### PR TITLE
Feature branch

### PR DESCRIPTION
**Pull request by karthykarthick**
Pull Request for the second project of the Microverse Program, which consists of duplicating the mint sign-up webpage using HTML and CSS.

### CODE REVIEW

**Comment by BrittanyBlake** [comment]
### Pull request under review by Brittany Blake (TSE)

File: index.html [hunk 1]
```diff
@@ -2,11 +2,127 @@
...
+              name="email"
+              
+              class="email"
+              required
+            />
+          </div>
+          <div class="info-block">
+            <label class="mail" >Phone (recommended)</label>
+            <input type="text" name="email" class="email" required />
+          </div>
```
Comment (BrittanyBlake) [inline index.html]: "phone" should have an `<input>`of type `tel`

**Review by BrittanyBlake — CHANGES_REQUESTED** [review]
### Status: Changes Required ♻️ 
****
### Hi @karthykarthick  and @abouhid! 👋 👋 
 You have also done a wonderful job with your project so far! 🎖️  It looks really close to the original page! 👏 👏 👏  There are just a few issues that need to be resolved before you can move on: ⬇️ 
****

- [x] The form should be wrapped in a `<form> tag`
Screen Shot 2020-06-11 at 6 57 49 PM

- [x] Please use more descriptive commit messages for your future commits. Commit messages such as "make some changes" should be more descriptive so one is able to identify what changes were made.

- [x]  Please add a more descriptive pull request that includes an appropriate title and a short summary of what you have done. 🗒️

- [x] Files not related to the project should not be committed and added to `.gitignore`. Please add the `.hintrc` file to `.gitignore.`

- [x] Please check my comments and make those changes as well.


****
### Please make the required changes and resubmit for a new code review once you are ready 🙏

Good Luck and Happy Coding 🍀 👩🏾‍💻

_P.S. If you have any questions about my review or need help, you can message me on slack (@ Brittany Blake)._

File: index.html (hunk 1 above)
Comment (abouhid) [reply index.html]: Issue corrected, thank you!

**Comment by abouhid** [comment]
We are sorry for the issue with the non-descriptive commits, should we try to edit them or just pay more attention in the future? We are going to be more aware of this flaw of ours

**Review by BrittanyBlake — APPROVED** [review]
### Status: Approved ✅ 
****
### Hi Team! 👋 
### Great job implementing the requested changes!! 🎖️ It's a 👍 from me! Go ahead and merge this pull request and keep up the good work 😄
****

_P.S. If you have any questions about my review or need help, you can message me on slack (@ Brittany Blake)._ 😄 

 Good Luck and Happy Coding! 🍀 👩🏾‍💻

**Pull request merged**